test/seqan3-test.cmake

```diff
@@ -19,6 +19,8 @@ include (CheckCXXSourceCompiles)
 include (FindPackageHandleStandardArgs)
 include (FindPackageMessage)
 
+option (SEQAN3_TEST_BUILD_OFFLINE "Skip the update step of external projects." OFF)
```

Comment:
What is this? When do I have to use/set it?

Reply:
Is this part of the fix?